server/models/Card.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose')
+
+const cardSchema = new mongoose.Schema({
+  cardTitle: {
+    type: String,
+    required: false
```

Comment:
`required: false` is the default behaviour of Mongoose. We don't need to explicitly define it. Please change where applicable.